pkg/oci/utils.go

```diff
@@ -220,11 +241,12 @@ func StatusToOCIState(status vc.PodStatus) (spec.State, error) {
 	}
 
 	state := spec.State{
-		Version: spec.Version,
-		ID:      status.ID,
-		Status:  stateToOCIState(status.ContainersStatus[0].State),
-		Pid:     status.ContainersStatus[0].PID,
-		Bundle:  status.ContainersStatus[0].RootFs,
+		Version:     spec.Version,
+		ID:          status.ID,
+		Status:      stateToOCIState(status.ContainersStatus[0].State),
+		Pid:         status.ContainersStatus[0].PID,
+		Bundle:      status.Annotations[ociBundlePathKey],
```

Comment:
This then would become `status.ContainerStatus[0].Annotations[ociBundlePathKey]`